README.md

```diff
@@ -60,7 +60,46 @@ Simply `npm install vue-fineuploader@next` and see the documentation for your sp
 
 #### `<gallery />`
 
-Not implemented yet.
+```javascript
+<template>
+  <Gallery :uploader="uploader" />
+</template>
+
+<script>
+  import FineUploaderTraditional from 'fine-uploader-wrappers';
+  import Gallery from '../modules/vue-fineuploader/src/gallery';
+
+  export default {
+    components: {
+      Gallery,
+    },
+    data () {
+      const uploader = new FineUploaderTraditional({
+        options: {
+          deleteFile: {
+            enabled: true,
+            endpoint: 'http://localhost:8000/uploads'
+          },
+          request: {
+              endpoint: 'http://localhost:8000/uploads',
+              // Used for AWS
+              // accessKey: 'JB/Ckd2bcqydVOYhbbAZo+kTgGWzeXQt9UacmaC7',
```

Comment:
I guess this shouldn't be there.

And, for the endpoint, you can use `my/upload/endpoint`, for consistency with the rest of the README